### PR TITLE
feat: add prettier-plugin-sh

### PR DESCRIPTION
**Pull request by jsjoeio**
This PR adds prettier-plugin-sh and removes shfmt.

_Note: it makes some minor modifications to the way our `.sh` files looked before. I tried setting some of the parser options based on [the docs](https://www.npmjs.com/package/prettier-plugin-sh#parser-options) but nothing seemed to work so left it as is. It uses the default options based on [mvdan/sh](https://github.com/mvdan/sh)_

### CODE REVIEW

File: ci/images/debian10/Dockerfile [hunk 1]
```diff
@@ -42,7 +42,6 @@ ENV PATH=/usr/local/go/bin:$GOPATH/bin:$PATH
 
 # Install Go dependencies
 ENV GO111MODULE=on
-RUN go get mvdan.cc/sh/v3/cmd/shfmt
 RUN go get github.com/goreleaser/nfpm/cmd/nfpm@v1.9.0
```
Comment (jawnsy) [inline ci/images/debian10/Dockerfile:45]: If we're able to drop `nfpm` next, does this mean we can drop the dependency on go?

**Review by jawnsy — DISMISSED** [review]
LGTM

File: ci/images/debian10/Dockerfile (hunk 1 above)
Comment (jsjoeio) [reply ci/images/debian10/Dockerfile:45]: I hope so! 🤞 tbh, I don't know what `nfpm` is used for, but hopefully we can drop that and Go soon! I'll add a note to the Clean Up issue to investigate

Comment (jawnsy) [reply ci/images/debian10/Dockerfile:45]: seems to be something release-related, I think we should clean up all of those images, particularly in light of the fact that CentOS changed their model recently with CentOS Stream

File: ci/lib.sh [hunk 2]
```diff
@@ -75,7 +75,7 @@ download_artifact() {
   local tmp_file
   tmp_file="$(mktemp)"
 
-  curl -fsSL "$(get_artifact_url "$artifact_name")" > "$tmp_file"
+  curl -fsSL "$(get_artifact_url "$artifact_name")" >"$tmp_file"
```
Comment (jawnsy) [inline ci/lib.sh:78]: just a personal preference, I like to use the long forms of the flags to make it clearer what's happening

File: ci/lib.sh [hunk 3]
```diff
@@ -2,11 +2,11 @@
 set -euo pipefail
 
 pushd() {
-  builtin pushd "$@" > /dev/null
+  builtin pushd "$@" >/dev/null
```
Comment (jawnsy) [inline ci/lib.sh:5]: I just might steal this code...

File: ci/images/debian10/Dockerfile (hunk 1 above)
Comment (jsjoeio) [reply ci/images/debian10/Dockerfile:45]: Good call. I'll add a note for that as well.

File: ci/lib.sh (hunk 3 above)
Comment (jsjoeio) [reply ci/lib.sh:5]: do it! who knows who wrote it 😂

**Review by jawnsy — APPROVED** [review]

File: ci/lib.sh (hunk 2 above)
Comment (jsjoeio) [reply ci/lib.sh:78]: I didn't write this 🤷‍♂️ but agree, I generally like things to be extra clear

**Comment by SuperSandro2000** [comment]
Whats the motivation behind this?

**Pull request merged**